x/mint/keeper/emissions.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	"github.com/allora-network/allora-chain/x/mint/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// return the uncirculating supply, i.e. tokens on a vesting schedule
+// latest discussion on how these tokens should be handled lives in ORA-1111
```

Comment:
Unfortunately others who access only to the open source code won't have access to ORA-1111, so is there another way we can write an informative comment?

Reply:
I hope that this comment will not survive the next 2 weeks and would be replaced before any outside eyes look at us seriously

that said, I can try to make this comment more informative now and remove the reference